spring-core-4/src/main/java/com/baeldung/dynamic/autowire/UkRegionService.java

```diff
@@ -0,0 +1,16 @@
+package com.baeldung.dynamic.autowire;
+
+import org.springframework.stereotype.Service;
+
+@Service("uk")
```

Comment:
since the names are global I would say this needs to be more specific, just uk seems a bad practice

Reply:
Ok, will change